### PR TITLE
Create Event Reporter project

### PR DESCRIPTION
**Pull request by VictoriaVasys**
* Based on the project found here: https://github.com/turingschool/curriculum/raw/master/source/projects/event_reporter.markdown

@AliSchlereth @mikedao Here's the updated markdown; I just added the rubric (with an additional version control portion, & changed method-length expectations to 7-lines  & expected enums to be best-choice) and fixed the link for the API in extensions.

I haven't had a chance to look it over carefully, but will do so later tonight. Let me know if anything stands out to you.

### CODE REVIEW

File: module1/projects/event_reporter.md [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+---
+
+This project builds on the lessons learned in [EventManager]({% page_url /projects/eventmanager %})
+to focus on fundamental Ruby style/concepts.
+
+## Project Overview
+
+### Learning & Practice Goals
+
+* Become comfortable with implementing basic classes and methods
```
Comment (AliSchlereth) [inline module1/projects/event_reporter.md]: I'd like to change this learning goal.  Perhaps this?
* Determine necessary classes and methods from project spec

I'm not sure that clearly describes what I'm going for. My idea is that they are practicing figuring out what classes and methods they need to build without having an interaction pattern to tell them what to do.

File: module1/projects/event_reporter.md [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+* Demonstrate understanding of variable scope and lifecycle
+* Create multiple coordinating methods and objects
+* Use default and named parameters
```
Comment (AliSchlereth) [inline module1/projects/event_reporter.md]: I'd like to take this one out and put in something about enumerables. Perhaps this:
* Choose enumerables specific to the task

File: module1/projects/event_reporter.md [hunk 3]
```diff
@@ -0,0 +1,405 @@
...
+* Utilize effective debugging techniques
+
+### Abstract
+
+Let's take [EventManager]({% page_url /projects/eventmanager %}) to the next
+level. Based on the same data file, build an interactive query and reporting
+tool which fulfills the expectations below. Re-use data cleaning procedures
+from the original `EventManager` to handle dirty input and generate beautiful
+output. We are also going to explore creating our own HTML files and
+using an API.
```
Comment (AliSchlereth) [inline module1/projects/event_reporter.md]: Maybe take out the `and using an API`

File: module1/projects/event_reporter.md [hunk 4]
```diff
@@ -0,0 +1,405 @@
...
+### 3. Test-Driven Development
+
+Expectations: 
+
+- [ ] Each method is tested  
+- [ ] Functionality is accurately covered
+- [ ] Tests implement Ruby syntax & style   
+- [ ] Balances unit and integration tests 
+- [ ] Evidence of edge cases testing 
+- [ ] Test Coverage metrics are present (SimpleCov)
```
Comment (AliSchlereth) [inline module1/projects/event_reporter.md]: Let's add a line for them to include a RakeTask too.
